src/picknik_ur_multi_arm_config/objectives/close_right_gripper.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <root BTCPP_format="4" main_tree_to_execute="Close First Gripper">
-    <BehaviorTree ID="Close Right Gripper" _description="Close the first gripper">
+    <BehaviorTree ID="Close First Gripper" _description="Close the first gripper">
```

Comment:
Apparently the `ID` has to be the same as `main_tree_to_execute` or the frontend will complain:
```
[web_server_node-1] Cannot parse Objective in "/home/michael/.config/moveit_pro/picknik_ur_multi_arm_config/objectives/close_right_gripper.xml: 409: Unable to read objective files /home/michael/.config/moveit_pro/picknik_ur_multi_arm_config/objectives/close_right_gripper.xml. Cannot retrieve the BehaviorTree entry.".
```